genesyscloud/outbound_contact_list_contact/resource_genesyscloud_outbound_contact_list_contact_utils.go

```diff
@@ -12,6 +12,7 @@ import (
 
 // buildWritableContactFromResourceData used to build the request body for contact creation
 func buildWritableContactFromResourceData(d *schema.ResourceData) platformclientv2.Writabledialercontact {
+	contactId := d.Get("contact_id").(string)
```

Comment:
if the user provides `null` for contact_id or completely omits it from the resource, I'm not sure that it will be an empty string by default. Either way, it definitely will be if you change this to 

```golang
contactId, _ := d.Get("contact_id").(string)
```
so it might be worth making that change just in case.

Reply:
@bbbco Can you make this small change.  Then we can merge

Reply:
Fixed